packages/manager/modules/manager-pci-common/src/api/data/regions.ts

```diff
@@ -49,3 +64,23 @@ export const addProjectRegion = async (
   });
   return data;
 };
+
+export const getProductRegionsAvailability = async (
+  ovhSubsidiary: string,
+  planCode: string,
+): Promise<string[]> => {
+  const { data } = await v6.get<TAvailabilityData>(
+    `/cloud/order/rule/availability?ovhSubsidiary=${ovhSubsidiary}&planCode=${planCode}`,
+  );
+  return data?.plans[0]?.regions;
+};
+
+export const getArchiveRegionDetails = async (
```

Comment:
this function is not related to archive it shoult be named getProjectRegionDetails